docs/kafka.md

```diff
@@ -0,0 +1,91 @@
+# `kafka` Metrics Receiver
+
+The `kafka` metrics receiver can fetch stats from a Kafka broker's Java Virtual Machine (JVM) via [JMX](https://www.oracle.com/java/technologies/javase/javamanagement.html).
+
+## Prerequisites
+
+In order to expose a JMX endpoint, you must set the `com.sun.management.jmxremote.port` system property. It is recommended to also set the `com.sun.management.jmxremote.rmi.port` system property to the same port. To expose JMX endpoint remotely, you must also set the `java.rmi.server.hostname` system property. These are often set by setting the environment variable "KAFKA_JMX_OPTS" in `bin/kafka-run-class.sh` and "JMX_PORT" in `bin/kafka-server-start.sh` for the Kafka service environment.
+
+## Configuration
+
+| Field                 | Default            | Description |
+| ---                   | ---                | ---         |
+| `type`                | required           | Must be `kafka`. |
+| `endpoint`            | `localhost:7199`   | The [JMX Service URL](https://docs.oracle.com/javase/8/docs/api/javax/management/remote/JMXServiceURL.html) or host and port used to construct the Service URL. Must be in the form of `service:jmx:<protocol>:<sap>` or `host:port`. Values in `host:port` form will be used to create a Service URL of `service:jmx:rmi:///jndi/rmi://<host>:<port>/jmxrmi`. |
```

Comment:
This .md says the default is :7199 but the code says :9999. Which one should actually be the default?